src/instructlab/training/tokenizer_utils.py

```diff
@@ -50,10 +59,12 @@ def setup_tokenizer(model_name_or_path, SPECIAL_TOKENS, CHAT_TEMPLATE):
         tokenizer.add_eos_token = False
 
     tokenizer.chat_template = CHAT_TEMPLATE
+    assert len(get_sp_token(tokenizer, SPECIAL_TOKENS.eos.token)) == 1
```

Comment:
We should either add a text message to this assertion error or throw a separate error such as `InvalidTokenizer` or something so that users know why this is failing, e.g.:

```suggestion
    assert len(get_sp_token(tokenizer, SPECIAL_TOKENS.eos.token)) == 1, "EOS token does not exist or is of incorrect length"
```

Reply:
Fixed this locally